app/controllers/requests_controller.rb

```diff
@@ -23,8 +23,9 @@ def edit
   # POST /requests
   def create
     @request = Request.new(request_params)
-
     if @request.save
+      Mailer.send_generic_email(false, @request.email, 'workshop.portal@gmail.com', I18n.t('requests.email.topic'),
```

Comment:
@cmfcmf That was my idea too, but after talking to @Baschdl's team I found out they don't really have a place for that yet and hardcode it overtime. I was hoping for good input on this here :) Do you have an idea where to put it? I guess it should be changeable in and organizer/admin view later.